### DISCUSSION TITLE
Version Control and Git

### DISCUSSION THREAD

**Discussion by rahul-lg** [community/community#68125] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

How can I revert a Git commit that introduced a critical bug without losing other changes?

**Comment by RijoKsd** [comment]
@rahul-lg  use `git revert <commit-hash>` , instead of  <commit-hash> use the hash value you got when git log

**Comment by Akash1134** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
